ray-operator/hack/local_deploy.sh

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+PROJECT_ROOT="${SCRIPT_DIR}/../.." # hack/ -> ray-operator/ -> kuberay/
+
+HELM_CHART_PATH="${PROJECT_ROOT}/helm-chart/kuberay-operator"
+
+RAY_OPERATOR_PATH="${PROJECT_ROOT}/ray-operator"
+
+HELM_RELEASE_NAME="kuberay-operator"
+
+# --- Configuration Variables ---
+IMAGE_TAG="${IMAGE_TAG:=kuberay-dev-tag}"
```

Comment:
Usually the tag name doesn't need to say `-tag`. The tag can be just `kuberay-dev`